### PR TITLE
Update security-privacy-questionnaire.md w/ latest questions.

### PR DESCRIPTION
**Pull request by chcunningham**
Pulling in new/revised questions from the editors draft of the
questionnaire.

### CODE REVIEW

**Comment by chcunningham** [comment]
FYI @drkron

File: security-privacy-questionnaire.md [hunk 1]
```diff
@@ -1,91 +1,133 @@
...
+leak information between modes.
 
-**Does this specification have a "Security Considerations" and
-"Privacy Considerations" section?**
+**Does this specification have both "Security Considerations" and "Privacy
+Considerations" sections?**
 
-[Yes](https://w3c.github.io/media-capabilities/#security-privacy-considerations).
+It has a combined section on Security and Privacy, mostly focused on Privacy.
+There are no known security impacts of the features in this specification.
```
Comment (mounirlamouri) [inline security-privacy-questionnaire.md:128]: Worth linking to it?

**Review by mounirlamouri — APPROVED** [review]

File: security-privacy-questionnaire.md (hunk 1 above)
Comment (chcunningham) [reply security-privacy-questionnaire.md]: Done.

**Comment by chcunningham** [comment]
Thanks Mounir

**Pull request merged**